lib/Doctrine/ORM/Tools/Console/Command/RunDqlCommand.php

```diff
@@ -120,7 +120,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $query->setMaxResults((int) $maxResult);
         }
 
-        if ($input->hasOption('show-sql')) {
+        if ($input->hasOption('show-sql') && $input->getOption('show-sql')) {
```

Comment:
Well, IMO, it is perfectly fine for a command to assume that the options it defined are still defined. If you want to allow anyone to hack the command, it should be done for all options and all arguments, and this does not make sense.
So I vote for remove the ``hasOption`` call